src/fontcache.cpp

```diff
@@ -23,11 +23,20 @@
 #include "safeguards.h"
 
 /** Default heights for the different sizes of fonts. */
-static const int _default_font_height[FS_END]   = {10, 6, 18, 10};
+static const int _default_font_height[FS_END]   = {10, 6, 16, 10};
 static const int _default_font_ascender[FS_END] = { 8, 5, 15,  8};
 
+const int _default_font_sane_max_height_limit = 40;
```

Comment:
Wrong formatting for a constant (you've formatted it as a global variable that can change), and could be less wordy.

```suggestion
const int DEFAULT_FONT_MAX_HEIGHT = 40; ///< The maximum height of the default font.
```

Reply:
I'll change it.  2 lines above this is:
"_default_font_height[FS_END]"
which is what I used as an example when I wrote those lines.  Is that line improperly formatted?